.travis.yml

```diff
@@ -25,6 +17,21 @@ jobs:
       addons:
         srcclr: true
 
+    - stage: 'Integration tests'
+      cache: false
+      language: minimal
+      env:
+        SDK=react
+        SDK_BRANCH=$TRAVIS_PULL_REQUEST_BRANCH
+        REPO_SLUG=optimizely/react-sdk-e2e-tests
```

Comment:
Please specify, it's other than conventional naming that's why specifying REPO_SLUG otherwise, we don't need it.